tests/test_define.h

```diff
@@ -332,6 +332,8 @@ extern "C" {
 #define MESSAGE_5 "MESSAGE5"
 #define MESSAGE_6 "MESSAGE6"
 
+#define TEST_PROXY_API_COMMAND "addNeighbors"
```

Comment:
Why would you specify the common name for single API?

Reply:
This is used for `test_serializer`. Or maybe I should declare a constant string in that test?

Reply:
How about renaming from `TEST_PROXY_API_COMMAND` to `TEST_PROXY_API_ addNeighbors`?